src/app/components/demo-components.service.ts

```diff
@@ -422,7 +422,7 @@ export class SkyDemoComponentsService {
         name: 'Tile',
         icon: 'th-large',
         // tslint:disable-next-line
-        summary: `The tile directive provides a container that is the building block for pages and forms.`,
+        summary: `The tile directive component creates a collapsible container that is a building block for pages and forms.`,
```

Comment:
The tile would be either a `component` or a `directive` but not both.  It appears the old version had `directive` a lot and I like the change to `component`. We should probably just remove the word `directive` here too.

Reply:
Agreed. No need for `directive`.

Reply:
Thanks for catching that. I removed the old instances of `directive` after confirming the change with Steve but apparently missed this one. Thanks!